wattsonic/wattsonic-registers.json

```diff
@@ -92,7 +92,7 @@
             "registerType": "holdingRegister",
             "description": "Phase A power",
             "defaultValue": 0,
-            "unit": "1/1000 kW",
+            "unit": "W",
```

Comment:
I remember I used 1/1000 kW because that's what their modbus registers documentation says... But sure, it's the same...

Reply:
saw that in the docs and thought...le's stop this non sese right here :P